superset_patchup/views.py

```diff
@@ -0,0 +1,61 @@
+"""
+Additional extensions to SuperSet views
+"""
+
+import json
+from flask_appbuilder import expose
+from flask_appbuilder.models.sqla.interface import SQLAInterface
+import superset
+
+from . import (VERSION, __version__)
+
+class SupersetKetchupApiView(superset.views.base.BaseSupersetView):
+
+    """
+    Extensions to Superset API
+    """
+
+    def __init__(self):
+        self.route_base = "/superset-ketchup/api"
```

Comment:
So I did some more digging on the /entity/api methods in superset - these look like they're actually default-generated endpoints from flask_appbuilder, and they're all getting deprecated, so I don't think we should follow that pattern.

This superset-ketchup/api seems sane to me, in that the idea is eventually when superset realizes the error of their ways and pulls all our patches, "superset-ketchup/api" endpoints will move to just "/api".

Reply:
Not 100% sure what you're asking, but the deprecation would just kill the default endpoints for a model view, and there is a separate api view that you can include optionally.

Here though we don't have default endpoints - we're not inheriting from that stuff.